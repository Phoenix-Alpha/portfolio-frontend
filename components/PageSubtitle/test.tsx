import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { PageSubtitle } from '.'

describe('<PageSubtitle />', () => {
  it('should renders correct <PageSubtitle />', () => {
    const wrapper = shallow(<PageSubtitle title="Page Subtitle" />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
