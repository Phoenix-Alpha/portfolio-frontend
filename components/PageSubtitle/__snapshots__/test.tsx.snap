// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageSubtitle /> should renders correct <PageSubtitle /> 1`] = `
<h2
  className="mb-6 font-body text-base lg:text-lg font-bold text-gray-2"
>
  Page Subtitle
</h2>
`;
